src/sql/workbench/contrib/notebook/browser/cellViews/codeCell.component.ts

```diff
@@ -99,6 +104,12 @@ export class CodeCellComponent extends CellView implements OnInit, OnChanges {
 
 	}
 
+	public toggleEditMode(): void {
+		this.isEditMode = this.isEditMode !== undefined ? !this.isEditMode : false;
```

Comment:
If `isEditMode` is undefined wouldn't toggling it be setting it to true? I would expect that undefined and false are the same thing here (but also why would we even have isEditMode be able to be undefined? That doesn't seem to make sense to me)

Reply:
isEditMode is never undefined